Lib/test/test_readline.py

```diff
@@ -9,14 +9,27 @@
 import sys
 import tempfile
 import unittest
-from test.support import import_module, unlink, temp_dir, TESTFN
+from test.support import import_module, unlink, temp_dir, TESTFN, verbose
 from test.support.script_helper import assert_python_ok
 
 # Skip tests if there is no readline module
 readline = import_module('readline')
 
 is_editline = readline.__doc__ and "libedit" in readline.__doc__
 
+
+def setUpModule():
+    if verbose:
+        # Python implementations other than CPython may not have
+        # these private attributes
+        if hasattr(readline, "_READLINE_VERSION"):
+            print(f"readline version: {readline._READLINE_VERSION:#x}")
+            print(f"readline runtime version: {readline._READLINE_RUNTIME_VERSION:#x}")
+        if hasattr(readline, "_rl_library_version"):
+            print(f"readline library version: {readline._rl_library_version!r}")
+        print(f"is libedit? {is_editline}")
```

Comment:
@berkerpeksag version showing "readline implementation: libedit|GNU readline" can be little nicer.